spec/memfs/file_system_spec.rb

```diff
@@ -289,6 +289,18 @@ module MemFs
         expect(subject.find!('/new-dir')).to be_a(Fake::Directory)
       end
 
+      it 'sets directory permissions to default 0777' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.